tools/runners/sv_parser.py

```diff
@@ -0,0 +1,15 @@
+from BaseRunner import BaseRunner
+
+
+class sv_parser(BaseRunner):
+    def __init__(self):
+        super().__init__("sv-parser", "parse_sv")
+
```

Comment:
If you add 
```
   self.url = "https://github.com/dalance/sv-parser"
```
.. the header link in the log-files will be pointing to your code.

Reply:
Thanks. I'll add it.